packages/terra-clinical-result/src/terra-dev-site/test/clinical-result/clinical-result-blood-pressure/SingleResultClinicalResultBloodPressure.test.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { ClinicalResultBloodPressure } from 'terra-clinical-result/lib/index';
+import { DefaultSystolicResult, DefaultDiastolicResult, NoDataResult, DefaultBloodPressureSystolicResultWithNoId, DefaultBloodPressureDiastolicResultWithNoId } from '../TestResults';
+
+const noDataDiastolic = {
+  systolic: {
+    ...DefaultSystolicResult,
+  },
+  diastolic: {
+    ...NoDataResult,
+  },
+};
+
+const noDataSystolic = {
+  systolic: {
+    ...NoDataResult,
+  },
+  diastolic: {
+    ...DefaultDiastolicResult,
+  },
+};
+
+const noData = {
+  systolic: {
+    ...NoDataResult,
+  },
+  diastolic: {
+    ...NoDataResult,
+  },
+};
+
+const ClinicalResultBloodPressureExample = () => (
+  <>
+    <ClinicalResultBloodPressure {...DefaultBloodPressureSystolicResultWithNoId} />
```

Comment:
Can we consume this like we normally would with props?

```suggestion
    <ClinicalResultBloodPressure
        systolic={DefaultSystolicResult}
        diastolic={NoDataResult}
    />
```

Reply:
Fixed https://github.com/cerner/terra-clinical/pull/614/commits/b0b5305327e8c17a49550348602d6ea79f39c368#diff-26fedcf401089021742523790c5c19a2R1